src/python3_pip_skeleton/__main__.py

```diff
@@ -17,12 +17,15 @@
 MERGE_BRANCH = "skeleton-merge-branch"
 # Extensions to change
 CHANGE_SUFFIXES = [".py", ".rst", ".cfg", "", ".toml"]
-# Files not to change
-IGNORE_FILES = [
-    "update-tools.rst",
-    "test_boilerplate_removed.py",
-    "pin-requirements.rst",
-]
+# Files not to change where IGNORE_FILES[x] is a list containing line numbers
+# in the file x to be ignored. An empty list will ignore the whole file.
+IGNORE_FILES: dict = {
```

Comment:
Nitpick: Is there a reason to specify `dict` here? I think it should either be a full type hint (`Dict[str, List[int]]`) or left blank and let the automatic type inference deal with it.

Reply:
I have deleted my comment having actually properly read what was written...

Reply:
We ned to typehint or the following is a mypy error:
``` Python
                for ignored_line in IGNORE_FILES[child.name]:
```
I'll switch to the full typehint.